sources/packages/api/src/route-controllers/student/student.aest.controller.ts

```diff
@@ -0,0 +1,36 @@
+import { Controller, Get, Injectable, Param } from "@nestjs/common";
+import { ApiTags } from "@nestjs/swagger";
+import { AuthorizedParties } from "../../auth/authorized-parties.enum";
+import { AllowAuthorizedParty, Groups } from "../../auth/decorators";
+import { UserGroups } from "../../auth/user-groups.enum";
+import BaseController from "../BaseController";
+import { StudentUploadFileDTO } from "./models/student.dto";
+import { StudentControllerService } from "./student.controller.service";
+
+/**
+ * Student controller for AEST Client.
+ */
+@AllowAuthorizedParty(AuthorizedParties.aest)
+@Groups(UserGroups.AESTUser)
+@Controller("students")
```

Comment:
devs, we never discussed this I guess, should this be singular or plural, i think our controller has a mix of both. thoughts?

Reply:
Just for the history, as we talked about in the call let's keep it singular.